src/Umbraco.Web/Models/Mapping/ContentMapperProfile.cs

```diff
@@ -88,12 +91,45 @@ internal class ContentMapperProfile : Profile
         }
     }
 
+    internal class CultureUpdateDateResolver : IValueResolver<IContent, ContentItemBasic<ContentPropertyBasic>, DateTime>
+    {
+        public DateTime Resolve(IContent source, ContentItemBasic<ContentPropertyBasic> destination, DateTime destMember, ResolutionContext context)
+        {
+            var culture = context.GetCulture();
+
+            //a culture needs to be in the context for a variant content item
+            if (culture == null)
```

Comment:
@warrenbuckley we can't throw here, culture can be null if we are mapping in an invariant context. If it's null it means that it's not been mapped with a culture, so we will have to assume there's no variants being worked on and we'd just return the source.UpdateDate.